comps/reranks/langchain-mosec/mosec-docker/server-ipex.py

```diff
@@ -0,0 +1,196 @@
+# Copyright 2024 MOSEC Authors
```

Comment:
can you pls check with @chensuyue if this license is correct?

Reply:
> can you pls check with @chensuyue if this license is correct?

Hi Feng,  I will clean up the code.

Reply:
Yes, please keep the 3rd party copyright.

Reply:
OK, I will keep the 3rd party copyright, and re-push the commit. Thanks for the confirmation.